sdk/dataprotection/azure-resourcemanager-dataprotection/CHANGELOG.md

```diff
@@ -1,14 +1,833 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.3.0 (2024-07-22)
 
-### Features Added
+- Azure Resource Manager DataProtection client library for Java. This package contains Microsoft Azure SDK for DataProtection Management SDK. Open API 2.0 Specs for Azure Data Protection service. Package tag package-2024-04. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
 
 ### Breaking Changes
 
-### Bugs Fixed
+#### `models.BackupInstanceResource` was modified
+
+* `suspendBackups(com.azure.core.util.Context)` was removed
```

Comment:
Is this breaking related with `x-ms-authorization-auxiliary` header?

Reply:
This breaking is just swagger change. In the version of `package-2024-04`, the following methods `models.BackupInstanceResource.suspendBackups/stopProtection` and
`models.BackupInstances.stopProtection/suspendBackups` added a parameter of type `Request`.

Reply:
Add important text such as: `- IMPORTANT: use `AuxiliaryAuthenticationPolicy` from "azure-resourcemanager-resources" module via `RecoveryServicesBackupManager.configure().withPolicy(..)`, for setting "x-ms-authorization-auxiliary" header.`